vault/dendron.tutorial.taking-notes.md

```diff
@@ -1,53 +1,91 @@
+
+Take a look at the Tree View to see the `vegetarian` note under the `recipes` hierarchy. You can also see the the note stored as `recipes.vegetarian.md` in the File Explorer Pane.
 
-> 🌱 To create a hierarchy, bring up lookup again `(Ctrl+L / Cmd+L)`. Type `recipes.vegetarian` and then press enter.
+#### Children
 
-You have now created your first hierarchy. Hierarchies in Dendron are created by having a `.` delimiter in the file name. Take a look at the tree view in the side panel to see that the `vegetarian` note exists under the `recipes` hierarchy. You can also see in the Workspace panel that the note file is stored as `recipes.vegetarian.md`.
+In this case, **vegetarian** is considered a child note of **recipes**. When looking at the `recipes` note, the preview view will list `vegetarian` (and any other `recipes.*` notes) at the bottom as **Children.** Children can also be seen and navigated via the tree view.
 
-You can create a hierarchy at any level:
+![[dendron://dendron.dendron-site/asset.preview#children,1:#*]]
 
-> 🌱 Type the following into lookup and hit enter: `recipes.italian.desserts.tiramisu`
+#### Hierarchy at any level
 
-You'll notice in the tree view that there is now a `+` sign next to `italian` and `desserts`. The plus sign indicates that this note is a stub. A stub is a placeholder for a note that hasn't actually been created. Dendron uses stubs to avoid cluttering your file system with empty notes when creating hierarchies.
+You can create a hierarchy at any level.
 
-![Tree View](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-tree-view.png)
+> 🌱 **ACTION:** Type the following into lookup and hit enter: `recipes.italian.desserts.tiramisu`
 
-Following this recipes example, you can try creating other notes that might be relevant. Some examples: `recipes.italian.appetizers`, `recipes.chinese`, `recipes.chinese.appetizers`, `recipes.ingredients.favorites`, `recipes.ingredients.shopping-list`. Hierarchies are flexible, and you can create them in whichever way works best for you. Dendron allows you to easily change the hierarchies later if you change your mind on how to organize your notes.
+You'll there is a `+` sign next to `italian` and `desserts` when viewing in the lookup or Tree View. The plus sign indicates that this note is a stub. A **[[stub|dendron://dendron.dendron-site/dendron.concepts#stubs]]** is a placeholder for a note that hasn't been created. Dendron uses stubs to avoid cluttering your file system with empty notes when creating hierarchies.
+
+![[dendron://dendron.dendron-site/asset.preview#tree-view-dark,1:#*]]
+
+Following this recipes example, you can create other related. Some examples:
+
+```
+recipes.favorites
+recipes.italian.appetizers
+recipes.chinese
+recipes.chinese.appetizers.wontons
+recipes.ingredients.favorites
+recipes.ingredients.shopping-list
+```
+
+You may wonder why Dendron uses hierarchies.
```

Comment:
this kind of breaks the flow of the tutorial and is making it take longer to go through. instead of a note reference, maybe have this as an aside so folks that are interested can followup?

> Aside: You may be wondering why Dendron uses hierarchies. You can [[read here|...]] for our thoughts about structure